cucumberjs/features/send_application_and_pay.feature

```diff
@@ -251,10 +251,6 @@ Feature: END TO END REGRESSION TEST
         When I click on "Continue" button
```

Comment:
I think you should have **Given**, **When** and **Then** used in that order and only once. It's fine to have **And** for multiple steps. I just think it reads a lot better, **Then** is really your assertion/thing that you want to test and you should only really be testing that one (or possibly two/three) things happen.

I know it would take a lot of work to refactor so I'm not saying do that now but I think a scenario should only really have 4 or 5 lines on it, the technical detail of what you are doing to drive the UI should be done in the step definition file (preferably using a method for that action that can change if the UI changes)

Reply:
I do agree that it should have a smaller scenario rather than a granular structure of each and every action. The only reason i am using this as I am using this for User research and Design screenshots required. Hence I added micro steps in. And unlike ruby, in js it doesn't allow to call steps inside steps.  will be reducing the steps to make it simpler to read. This will be done when we will have some time to tidy and refactor previous code. While for new tests addition, I will improvise to reduce the granularity